recipe/meta.yaml

```diff
@@ -12,33 +12,37 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 2
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python >=3.7
     - setuptools
-    - docutils >=0.8
+    - docutils >=0.8,!=0.18.*,!=0.19.*
     - pybtex >=0.24
     - pybtex-docutils >=1
-    - sphinx >=2.1
+    - sphinx >=3.5
     - dataclasses  # can be removed when cf drops py3.6 support
     - importlib_metadata >=3.6
   run:
-    - python >=3.6
-    - docutils >=0.8
+    - python >=3.7
+    - docutils >=0.8,!=0.18.*,!=0.19.*
     - pybtex >=0.24
     - pybtex-docutils >=1
-    - sphinx >=2.1
+    - sphinx >=3.5
     - dataclasses  # can be removed when cf drops py3.6 support
     - importlib_metadata >=3.6
 
 test:
+  requires:
+    - pip
   imports:
     - sphinxcontrib
     - sphinxcontrib.bibtex
+  commands:
+    - pip check
```

Comment:
Added this `pip check` based on @bollwyvl 's suggestion here: https://github.com/conda-forge/sphinxcontrib-bibtex-feedstock/pull/25#issuecomment-1693753935